doc(install): update ITEE installation page (#3109)
centreon/centreon-documentation: versioned_docs/version-24.04/administration/custom-login.md

- **Login page:** you can change the background and the logo, and add a text.
- **Name of the platform:** you can define a name for your platform (e.g. Test, Production) that will be displayed in the banner once you are logged in.

> To be able to customize your Centreon platform, you need to use the **Centreon IT Edition Extensions** extension. This extension is automatically installed on your platform.

Reviewer: ```suggestion
Customizing your Centreon platform required the **Centreon IT Edition Extensions** extension. This extension is automatically installed on your platform.
```

> **Centreon IT Edition Extensions** is a Centreon extension that requires a valid [license](../administration/licenses.md).
> To purchase one, contact [Centreon](mailto:sales@centreon.com).

## Configure the extension

1. Go to **Administration > Customize Centreon**. The different options are:
   

>    - **Platform name**: define a name for your platform (e.g. Test**, **Production). This name will be visible both on the login page (next to the logo), and in the banner once you are logged in.

Author: strange stars are not going well ***

>    - **Platform name**: define a name for your platform (e.g. Test**, **Production). This name will be visible both on the login page (next to the logo), and in the banner once you are logged in.

Reviewer: ```suggestion
   - **Platform name**: define a name for your platform (e.g. **Test**, **Production**). This name will be visible both on the login page (next to the logo), and in the banner once you are logged in.
```
